### ISSUE TITLE
Train MobileNetV2 with ImageNet

### ISSUE THREAD

**Issue by amandayeyan** [KaiyangZhou/deep-person-reid#208]
Hi Kaiyang, Thanks for the great sharing. Can you also share how you trained MobileNetV2 (specifically for person-reid application) with ImageNet?
What are the data pre-process steps for training? Did you use the same data pre-process as in the person-reid data transform ?

**Comment by KaiyangZhou** [comment]
random crop + flip, normalisation with imagenet mean and std, batch size 2048 with 8 gpus

the training code is from https://github.com/switchablenorms/Switchable-Normalization

**Comment by amandayeyan** [comment]
resize to 256x128 or 224x224?

**Comment by KaiyangZhou** [comment]
224x224 for imagenet

**Comment by amandayeyan** [comment]
So all the pretrained image model provided by you were trained with 224x224? 256x128 resolution only applies to funetune person reid model?

**Comment by KaiyangZhou** [comment]
> So all the pretrained image model provided by you were trained with 224x224? 256x128 resolution only applies to funetune person reid model?

Yes
